src/Microsoft.AspNetCore.Razor.Design/project.json

```diff
@@ -12,7 +12,10 @@
       "taghelper",
       "aspnetcore",
       "taghelpers"
-    ]
+    ],
+    "files": {
+      "include": [ "buildCrossTargeting/*", "build/*" ]
```

Comment:
So in this case won't the `ResolveRazorTargetFramework` be called multiple times if the project targets more than TFM?